apps/foundation-app/src/app/cart/components/proceed-to-checkout/proceed-to-checkout.component.html

```diff
@@ -0,0 +1 @@
+<div class="button">proceed to checkout</div>
```

Comment:
@lderrickable Even though we're keeping the component, I think we can still use foundation's button styles and eliminate the duplicate styling?

Reply:
Oh, I'm planning on changing it. I just ran out of time

Reply:
Okay. Thanks!

Reply:
Oh, I just saw Damien's comment. But anyway, it turned out that there were some styling conflicts between the foundation button class and the class I made that toggled `display: none`, so it would've needed to be wrapped anyway

Reply:
Ok, I made the styling changes